src/features/events/components/EventsList/index.tsx

```diff
@@ -12,15 +12,23 @@ import { useEvents } from '../../hooks/useEvents'
 import type { Event } from '../../types'
 
 const sorts = {
-  ascending: (a: Event, b: Event) => (a.startsAt < b.startsAt ? -1 : 1),
-  descending: (a: Event, b: Event) => (a.startsAt > b.startsAt ? -1 : 1),
+  asc: (a: Event, b: Event) => (a.startsAt < b.startsAt ? -1 : 1),
+  desc: (a: Event, b: Event) => (a.startsAt > b.startsAt ? -1 : 1),
```

Comment:
I think `asc` and `desc` are good short descriptors for these methods... it will help reduce line-breaks that could happen due to long lines :)